fed/mosaic-sns/src/main/java/org/eclipse/mosaic/fed/sns/ambassador/SnsAmbassador.java

```diff
@@ -75,12 +75,13 @@
      */
     public SnsAmbassador(AmbassadorParameter ambassadorParameter) {
         super(ambassadorParameter);
-        log.info("Start Simulation");
     }
 
     @Override
     public void initialize(final long startTime, final long endTime) throws InternalFederateException {
         super.initialize(startTime, endTime);
+        this.log.info("Init simulation with startTime={}, stopTime={}", startTime, endTime);
```

Comment:
better call it `endTime`

Reply:
done
https://github.com/fhilg/mosaic/commit/a6a043557a81ce10b87586eccd5dc9c23c792c3b